automated-comments/javascript/general/unexpected_splat_args.md

```diff
@@ -0,0 +1,7 @@
+Define a parameter called `%<splat-arg.name>s` with the type `%<parameter.type>s`,
+instead of using `...%<splat-arg.name>s: %<parameter.type>s[]`.
+
+[Rest parameters / splat arguments](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters)
+are great if the function doesn't dictate how many values it wants to receive,
+but in this case it is clearly defined how many values the function will be
```

Comment:
```suggestion
but in this case it is explicitly defined how many values the function will be
```

This sound a bit more, eh, friendly to me. Clearly almost implies a bit of a judgment, which could be interpreted as: "how could you _not_ see that!" :)

Reply:
Agreed! I can _clearly_ see how that could be the case now that you say it

Reply:
Ooh, I like what you did there  😄